packages/langium/src/test/langium-test.ts

```diff
@@ -17,17 +17,23 @@ import { findNodeForProperty } from '../utils/grammar-util';
 import { SemanticTokensDecoder } from '../lsp/semantic-token-provider';
 import { TextDocument } from 'vscode-languageserver-textdocument';
 import { expect } from 'vitest';
-import { BuildOptions } from '../workspace/document-builder';
 
-export function parseHelper<T extends AstNode = AstNode>(services: LangiumServices): (input: string, buildOptions?: BuildOptions) => Promise<LangiumDocument<T>> {
+export interface ParseHelperOptions {
+    validationChecks?: boolean;
+    documentUri?: string;
+}
```

Comment:
Why not extend the `BuildOptions` to avoid a breaking change and allow full flexibility (we might extend the BuildOptions in the future)?

Reply:
Note that this wouldn't have been a breaking change, since the `BuildOptions` argument has been introduced in a recent PR that hasn't been released yet. I have adopted your suggestion anyway, to improve flexibility.